pulp_rpm/app/modulemd.py

```diff
@@ -65,15 +65,16 @@ def split_modulmd_file(file):
     m_open = (
         gzip.open if file.url.endswith(".gz") else lzma.open if file.url.endswith(".xz") else open
     )
-    with m_open(file.path, "r") as modulemd_file:
+    with m_open(file.path, "rt") as modulemd_file:
```

Comment:
But I thought [text mode is the default](https://docs.python.org/3/library/functions.html#open), so why would this make a difference?

Reply:
Because for gzip and lzma.open() "r" is binary-mode:

https://docs.python.org/3/library/gzip.html#gzip.open
https://docs.python.org/3/library/lzma.html#lzma.open

'The mode argument can be any of "r", "rb", "w", "wb", "x", "xb", "a" or "ab" **for binary mode**, or "rt", "wt", "xt", or "at" for text mode. The default is "rb".'

Reply:
well that is quite frustrating.

Let's add a comment noting this